tests/gui/models/selection-state.spec.js

```diff
@@ -403,20 +433,62 @@ describe('Browser: SelectionState', function() {
           }).to.throw('Missing image size');
         });
 
-        it('should throw if size is not a number', function() {
+        it('should throw if the original size is not a number', function() {
+          m.chai.expect(function() {
+            SelectionStateModel.setImage({
+              path: 'foo.img',
+              size: {
+                original: '999999999',
+                final: {
+                  estimation: false,
+                  value: 999999999
+                }
+              }
+            });
+          }).to.throw('Invalid original image size: 999999999');
```

Comment:
I did a quick search and found https://gist.github.com/JacobOscarson/734620 but I agree this probably isn't worth changing.